src/HealthChecks.ClickHouse/HealthChecks.ClickHouse.csproj

```
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.1;net6.0;net7.0;net8.0</TargetFrameworks>
```

6.0 is out of support, we need to target net8.0

https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/pull/2331

```suggestion
    <TargetFrameworks>netstandard2.1;$(DefaultNetCoreApp)</TargetFrameworks>
```